techniques/general/G157.html

```diff
@@ -113,15 +110,15 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
                 </li>
             <li>
                   <a href="http://www.captionedtext.com/">Captioned Text</a>
-                  (provider for relay conference captioning)</li>
+                  (リレー会議字幕プロバイダー)</li>
             <li>
                   <a href="https://en.wikipedia.org/wiki/Communication_Access_Real-Time_Translation">Wikipedia entry for CART</a>
-                  (variously Computer Assisted Real-time Captioning or Communication Access Real-time Translation)</li>
+                  (コンピュータ支援によるリアルタイム字幕作成、通信アクセス、リアルタイム翻訳など)</li>
```

Comment:
CART は Communication Access Real-time Translation の略語のようです。「Computer Assisted Real-time Captioning」すなわち「Communication Access Real-time Translation」というニュアンスなのかなと思います。
```suggestion
                  (さまざまなコンピュータ支援によるリアルタイム字幕作成、すなわち通信アクセスのリアルタイム翻訳)</li>
```

Reply:
ここのorは「又は」が適切では…？
```suggestion
                  (さまざまなコンピュータ支援によるリアルタイム字幕作成、又は通信アクセスのリアルタイム翻訳)</li>
```